pom.xml

```diff
@@ -25,7 +25,7 @@
   <parent>
     <groupId>com.xwiki.parent</groupId>
     <artifactId>xwikisas-parent-platform</artifactId>
-    <version>13.10-3</version>
+    <version>14.10</version>
```

Comment:
Side note, please don't forget to create an issue for adding Licensing to this application